dev-tools/notice/overrides.json

```diff
@@ -1,3 +1,5 @@
+{"name": "github.com/leehinman/spigot", "licenceType": "Elastic"}
+{"name": "github.com/elastic/elastic-agent-shipper-client", "licenceType": "Elastic"}
```

Comment:
How did this become a dependency?

Reply:
The shipper dep is due to https://github.com/leehinman/spigot/blob/main/go.mod.